Source/Redemption/Private/RedemptionPlayer.cpp

```diff
@@ -244,3 +248,17 @@ void ARedemptionPlayer::UpdateStaminaWidget(float StamPercent)
 {
 	// Will affect the HUD elements eventually.
 }
+
+void ARedemptionPlayer::LauchPauseMenu()
+{
+	UPauseMenuWidget* UserInterface = CreateWidget<UPauseMenuWidget>(GetWorld(), PauseWidgetClass);
+	UserInterface->AddToViewport();
+	UGameplayStatics::SetGamePaused(GetWorld(), true);
+	APlayerController* PlayerController = UGameplayStatics::GetPlayerController(GetWorld(), 0);
+	PlayerController->SetInputMode(FInputModeUIOnly());
+	PlayerController->bShowMouseCursor = true;
```

Comment:
you are already doing this in the native constructor for the pause widget, you can remove one of them